assignments/assignment01.rb

```diff
@@ -68,7 +76,13 @@ def my_reverse(s)
 # Write a method `palindrome?`
 # that determines whether a string is a palindrome
 def palindrome?(s)
-  # your implementation here
+  forward = s.downcase.delete(" ").delete(",")
+  backward = s.downcase.reverse.delete(" ").delete(",")
+    if forward == backward
```

Comment:
you don't want to print out the result, you want to return the result. So, of forward == backward, you want to return true; false otherwise. The simplest way to do this is to just use the statement "forward == backward" as the return value.